File: Python/섹션 3 - (진행중) 탐색&시뮬레이션(string,1차원,2차원 리스트 탐색)/1. 회문 문자열 검사 (중)/AA.py
import sys
#sys.stdin=open('input.txt', 'rt')

# 개수 입력
n=int(input())

# 문자열 여러 줄 입력 [신기술]
arr=[input() for _ in range(n)]

for i in range(len(arr)):
    temp=''
    for c in arr[i]:
        temp=c+temp

    # 형식 출력    
    print('#%d'%(i+1), end=' ') 
    
    # 대소문자 구분 제거
    if arr[i].lower() == temp.lower():
        print('YES')
    else:
        print('NO')

File: Javascript/섹션3. (진행중) 문자열 탐색/1. 회문 문자열 (하)/AA.js
function solution(str){
    let temp=''
    let p=''
    
    str=str.toLowerCase()
    for(e of str) {
        temp=e+temp
    }
    if(str==temp){
        p=('YES')
    }
    else{
        p=('NO')
    }
    return p
}
let str="2gooG2"
console.log(solution(str))